website_search.py

```diff
@@ -0,0 +1,36 @@
+import requests
```

Comment:
This file looks like it needs more clean up - you can exclude this file from this PR?

Also, it's not clear what this file does - could you leave some comments at the top of this file, describing what this file does? It's also a good idea to leave some comments in *every* file - since all the files in the crawler isn't some standard framework code.

Reply:
It means that PR is merge to master????